packages/react/src/useEditor.ts

```diff
@@ -124,3 +124,17 @@ export const useEditor = (options: Partial<EditorOptions> = {}, deps: Dependency
 
   return editorRef.current
 }
+
+export const useEditor = (options: Partial<EditorOptions> = {}, deps: DependencyList = []): Editor | null => {
+  const editor = useEditorBase(options, deps)
+
+  return editor
+}
+
+// In case you want to render the editor immediately, you can use this hook.
+// Caution: Does not work with SSR.
+export const useEditorForImmediateRender = (options: Partial<EditorOptions> = {}, deps: DependencyList = []): Editor => {
+  const editor = useEditorBase(options, deps, true) as Editor
```

Comment:
This a nice way to abstract the base editor logic unlike my PR. However, I highly recommend using memo instead of ref as the lifecycle becomes a lot more simpler (especially with editor cleanup)